exotic/inputs.py

```diff
@@ -1,7 +1,7 @@
 import logging
-import json
 import sys
 from pathlib import Path
+from json import load
```

Comment:
to nitpick- why do a `from jason import load` rather than `import json` and then later on `json.load(json_file`? The only reason I'm pushing back on this is that there might be another use of the word "load" either today, or perhaps in the future, and calling it this other way might be safer in the long term

Reply:
I can us adding more methods in the future from the JSON module. Restored it to it's previous state.